cmd/aperture-agent/config/types.go

```diff
@@ -58,9 +58,9 @@ type AgentOTelConfig struct {
 	otelconfig.CommonOTelConfig `json:",inline"`
 	// DisableKubernetesScraper disables metrics collection for Kubernetes resources.
 	DisableKubernetesScraper bool `json:"disable_kubernetes_scraper" default:"false"`
-	// BatchPrerollup configures batch_prerollup processor.
+	// BatchPrerollup configures batch pre rollup processor.
```

Comment:
To resolve these issues you can use code block. e.g. `batch_prerollup`. Though in this case it makes sense to provide a more descriptive comment.

Reply:
Changed the description